scripts/data-import/judge/bulkImportJudgeUsers.helpers.js

```diff
@@ -39,7 +39,7 @@ const init = async (csvFile, outputMap) => {
           } else {
             const services = await getServices();
             endpoint = `${
-              services[`gateway_api_${process.env.DEPLOYING_COLOR}`]
+              services[`gateway_api_${process.env.CURRENT_COLOR}`]
```

Comment:
This is breaking deploys to experimental environments, I believe we want to import judge users into the `deploying_color` so we can run the smoketests with judges populated. Was this an accidental commit during debugging?